Resources/Steps/ApiSteps.robot

```diff
@@ -0,0 +1,87 @@
+*** Settings ***
+Resource  ../Tests_Imports.robot
+
+Library    Collections
+
+*** Variables ***
+
+
+*** Keywords ***
+Get Response From GET Weather request
+    [Documentation]  Send GET /weather request And Get Its Response
+    [Arguments]  ${params}
+    ${response} =    GET  ${API_URL}/weather  params=${params}
+    Status Should Be    OK    ${response}
+    [Return]  ${response}
+
+Get Weather Of The City By Name
+    [Arguments]  ${city}
+    ...          ${units}=${EMPTY}
+    IF  "${units}" == "${EMPTY}"
+        ${params} =    Create Dictionary    q=${city}
+        ...                                 appid=${API_KEY}
+    ELSE
+        ${params} =    Create Dictionary    q=${city}
+        ...                                 appid=${API_KEY}
+        ...                                 units=${units}
+    END
+    ${response} =    Get Response From GET Weather request  params=${params}
+    ${city_weather} =  Convert To Dictionary  ${response.json()}[main]
+    [Return]  ${city_weather}
+
+Get Temperature Of The City By Name
+    [Arguments]  ${city}
+    ...          ${units}=${EMPTY}
+    ${city_weather} =  Get Weather Of The City By Name  city=${city}
+    ...                                                 units=${units}
+    [Return]  ${city_weather}[temp]
+
+Get Longitude And Latitude From Get Weather Request With Name Param
+    [Arguments]  ${city}
+    ${params} =    Create Dictionary    q=${city}   appid=${API_KEY}
+    ${response} =    Get Response From GET Weather request  params=${params}
```

Comment:
looks like `Get Weather Of The City By Name`

Reply:
Hm, they are different:
`Get Weather Of The City By Name `- here we get weather
`Get Longitude And Latitude From Get Weather Request With Name Param` - here we get longitude and latitude